frontend/src/components/DriverCards/DriverCards.tsx

```diff
@@ -109,6 +103,11 @@ const DriverCards = () => {
       ))}
     </div>
   );
-};
+  };
+const DriverCards = () => (
+  <DriversProvider>
+    <Drivers/>
+  </DriversProvider>
```

Comment:
instead of putting the provider around every component that uses the context, you can wrap the entire app with the provider in `App.tsx`, which removes the need to create/rename existing components